hack/lib/etcd.sh

```diff
@@ -151,7 +151,7 @@ kube::etcd::install() {
       ln -fns "etcd-v${ETCD_VERSION}-darwin-amd64" etcd
       rm "${download_file}"
     elif [[ ${os} == "linux" && ${arch} == "amd64" ]]; then
-      url="https://github.com/centaurus-cloud/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-${os}-${arch}.tar.gz"
+      url="https://github.com/coreos/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-${os}-${arch}.tar.gz"
```

Comment:
Nothing found in https://github.com/etcd-io/etcd/releases/download/v3.4.3/

Reply:
> https://github.com/coreos/etcd/releases/download

Go to https://github.com/coreos/etcd/releases/download, page not found.

Reply:
Coreos was moved to etcd-io. New binary download should be:
https://github.com/etcd-io/etcd/releases/download/v3.4.3/etcd-v3.4.3-linux-amd64.tar.gz

https://github.com/coreos/etcd is redirected to https://github.com/etcd-io/etcd